scripts/build_dependencies/build_step_tar.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+ const process = require("process");
+ const build_step_axios = require("./build_step_axios");
+ const utils = require('./build_utils');
+
+module.exports = {
+
+    tar : null,
+
+    /**
+     * Will download the file at the given url with the given version to the given path using tar.
+     * Will automatically either use or download the runtime-package for tar as needed.
+     */
+    performStep: async function (url, version, path) {
+        if (utils.npmCheckIfPackageExists("tar")) {
+            await this.buildSource(url, version, path);
```

Comment:
I didn't see this function here. Is it missed or did I misunderstand it?

Reply:
Yeah, this was a copy-paste error 😅 
I'll see if I can test it locally to make sure it is working as expected 👍

Reply:
Tested and confirm it works now 👍